cmd/boot-script-service/sm.go

```diff
@@ -59,8 +59,8 @@ type SMComponent struct {
 }
 
 type SMData struct {
-	Components []SMComponent                  `json:"Components"`
-	IPAddrs    map[string]sm.CompEthInterface `json:"IPAddresses"`
+	Components []SMComponent                    `json:"Components"`
+	IPAddrs    map[string]sm.CompEthInterfaceV2 `json:"IPAddresses"`
```

Comment:
just to be clear; I take it this struct def already existed in the package as you didnt have to update the go.mod?

Reply:
FWIW the `CompEthInterfaceV2` structure has existed in HSM for about a year or so at this point.